gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Fri May 01 19:39:41 CEST 2020
+#Thu Oct 15 11:41:05 CEST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip
```

Comment:
Why `6.5`? Use `6.7` directly, I'd say, so that we don't have to upgrade again

Reply:
No, this is the version suggested by Android Studio. AGP is usually a bit behind on Gradle versions.